rules/dropbox_rules/dropbox_external_share.yml

```diff
@@ -0,0 +1,152 @@
+AnalysisType: rule
+Description: Dropbox item shared externally
+DisplayName: "Dropbox External Share"
+Enabled: true
```

Comment:
If allowed_domains is defined with example.com, this should probably be enabled:false and needs a `configuration required` tag.